cluster/config-defaults.yaml

```diff
@@ -1146,3 +1146,11 @@ control_plane_graceful_shutdown: "true"
 #  fs.aio-max-nr = 8388608
 #  fs.inotify.max_user_watches = 100000
 sysctl_settings: ""
+
+# enables/disables the minDomains field for pod topology spread.
+min_domains_in_pod_topology_spread_enabled: "true"
```

Comment:
This field doesn't seem to be used anywhere?

Reply:
seem to be a leftover from rebasing. removed it now

Reply:
This feature became GA in `1.31` so we got rid of this config-item